setup.py

```diff
@@ -22,7 +22,10 @@
 except ImportError:
     from distutils.core import setup
 
-import sys, os
+import setuptools
```

Comment:
Well I left import setuptools, but it seems to be unused... Thanks for review :)